README.md

```diff
@@ -40,64 +40,141 @@
+
+Install [lerna](https://github.com/lerna/lerna). Lerna is a tool that optimizes the workflow around managing multi-package repositories with git and npm. 
 
 ```sh
 npm i lerna -g
 ```
 
-Then you should install dependencies using the bootstrap command.
+Install dependencies: 
 
 ```sh
 lerna bootstrap
 ```
 
-## Development
+Create your env file:
 
-### Beta app
+```sh
+cp beta/.env.example beta/.env
+```
+
+You may point your `.env` file to the production API:
+
+```
+API_DOMAIN=api.resonate.coop
+APP_DOMAIN=beta.stream.resonate.coop
+APP_HOST=https://beta.stream.resonate.coop
+APP_PORT=""
+```
 
-To start beta app with `bankai`.
+Run the app:
 
 ```sh
-npm start
+npm run dev
 ```
 
-By default, the app is accessible at `https://localhost:8080`.
+You should see the app running at http://localhost:8080.
 
 You can run the app at a different port using the command below:
 
 ```sh
 npm run dev -- -- --port 8089
 ```
 
-See [bankai](https://github.com/choojs/bankai) docs for usage.
+The embed app (optional)
 
-### Other apps (embed, upload, ...)
+```sh
+lerna run --scope embed start --parallel
+```
+
+### Using HTTPS
+
+You will notice the app does not work well without HTTPS. Secure connection is required to fully test PWAs.
+
+#### Nginx
+
+You can find a reference [nginx configuration](/docs/nginx/beta.resonate.localhost.conf) file in the docs.
+Note that the reference is not a complete `nginx.conf` file, it should fit within your existing configuration or be wrapped in a http block directive:
+
+```
+http {
+  server {
+  }
+}
+```
 
-Replace `embed` with the target app if different. 
+In the example `nginx.conf`, note the lines referring to the `ssl_certificate_key` and the `ssl_certificate_key`. 
+
+#### Generating a Custom Certificate
+
+You can generate a custom certificate using [mkcert](https://github.com/FiloSottile/mkcert) for `beta.resonate.localhost`. *This origin is whitelisted in our CORS config*.
+
+Generate the certificate (`cert.pem`) and key (`key.pem`) for `beta.resonate.localhost`:
 
 ```sh
-lerna run --scope embed start --parallel
+mkcert -key-file key.pem -cert-file cert.pem beta.resonate.localhost
 ```
 
+In your nginx.conf file, update the `ssl_certificate_key` and the `ssl_certificate_key` to refer to your new key and certificate files. 
+
+#### Update your Hosts file
+
+Update your hosts file to include:
+
+```
+127.0.0.1       beta.resonate.localhost
+```
+
+#### Update on your .env file
+
+```sh
+APP_DOMAIN=beta.resonate.localhost
+APP_HOST=https://beta.resonate.localhost
+```
+
+#### Run the app!
+
+```sh
+npm run dev
+```
+
+You should now see the player running on https://beta.resonate.localhost
+
+## Development
+
+### API
+
+If you want to build on the API for personal use, take a look through the work done in the [Migrate to new APIs project](https://github.com/resonatecoop/stream/projects/5#card-59829409) and consider checking the [backlog in our community forum](https://community.resonate.is/c/platform/52). 
```

Comment:
The reference to the "Migrate to the new APIs project" will be soon out of date. Let's remove that part.

Reply:
removed